<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Continue</class>
 <widget class="QMainWindow" name="Continue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0.435644, y1:0.767, x2:1, y2:0, stop:0 rgba(20, 19, 5, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Above_Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>The information has been saved.</string>
    </property>
   </widget>
   <widget class="QLabel" name="Below_Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Would you like to continue?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="New_Button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NEW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
